INSTRUCTIONS.md

```diff
@@ -0,0 +1,97 @@
+# Using Anaconda
+
+IMPACT-T is available through conda-forge and can be installed via:
+```bash
+conda create -n impact
+source activate impact # or conda activate impact
+# For non-MPI
+conda install -c conda-forge impact-t
+
+# For OpenMPI
+conda install -c conda-forge impact-t=*=mpi_openmpi*
+
+# For MPICH
+conda install -c conda-forge impact-t=*=mpi_mpich*
+```
+# Compiling The Code
+
+## Unix
+
+### Single Processor Code:
+
+```shell script
+mkdir build
+cd build
+cmake ../src
+make
+make install
+```
+
+### Multi Processor Code:
+
+```shell script
+mkdir build
+cd build
+cmake ../src -DUSE_MPI=ON
+make
+make install
+```
+
+## Windows
+
+### Single Processor Code:
+
+```shell script
+mkdir build
+cd build
+cmake -G "NMake Makefiles" ../src
+cmake --build .
```

Comment:
In this case, shouldn't it be:
```
cmake -S ./src -B build -G "NMake Makefiles"
...
```

Since there is no top-level CMakeLists.txt and the only one lives inside of `<>/src`?

Reply:
Addressed via https://github.com/impact-lbl/IMPACT-T/pull/15/commits/0761cf88063ab37c739070c79ce46f6034a68a23